tests/dashbio_demos/app_needle_plot.py

```diff
@@ -103,108 +103,260 @@ def description():
 
 def header_colors():
     return {
-        'bg_color': '#009900',
-        'font_color': '#DFE8F3',
-        'light_logo': False
+        'bg_color': '#0D1A51',
+        'font_color': '#FFF',
```

Comment:
```suggestion
        'font_color': '#FFFFFF',
```

Reply:
Don't those two mean the same thing?

Reply:
Yes, in a minimalistic perspective, I can understand why you'd keep only 3 F's, since they're enough (to work). But, in a standard-oriented perspective, I don't know why we wouldn't go for the standard hexadecimal notation...